### DISCUSSION TITLE
Projects Beta: Notes / Issue Drafts, minor UX feedback

### DISCUSSION THREAD

**Discussion by lsl** [community/community#5822] [Projects and Issues]
Overall very happy with the new projects beta, these are just some minor issues I've run into using issue drafts:

1. ~Adding a new note opens some kind of editor, while double clicking on one edits in line. Now that you're scrolling to where it will be added (thank you!) you can open the in line editor straight away, no need for the secondary popup editor at the bottom of the page.~ I take this back, I see the editor stays open until Esc or something is hit, this lets you type many new notes one after the other, this is quite nice / worth the trade off.
 
2. ~Shift/Ctrl+Enter doesn't add a new line, in fact new lines are stripped from content, and there is no `markdown` ability, this is a big step back from being able to copy paste logs and stack traces into these things which is commonly the first thing that goes into a new issue. I'm guessing this is being forced by table view or something but I think that's quite fixable with truncation or treating anything after the first new line as body content when on the table view.~ The addition of being able to edit draft issue content has alleviated this significantly.

3. The "Add item" button has moved from the top of the column to the bottom, this might just be aversion to change but on a full sized desktop screen it may as well be below the fold.

**Comment by pshriwise** [comment]
Adding a small item here. It would be nice to be able to save issue drafts with ctrl+Enter as you would for most other fields in GH (proper issues, comments, this discussion box, etc.)
